docs/app/Examples/elements/Button/Variations/Colored.js

```diff
@@ -0,0 +1,11 @@
+import React, {Component} from 'react';
+
+export default class extends Component {
+  render() {
+    return (
+      <div className='ui warning message'>
+        TODO
```

Comment:
I noticed this example was missing, so I added a filler "TODO" example so we don't forget it.